project/Dependencies.scala

```diff
@@ -392,18 +392,19 @@ object Dependencies {
   val `testkit-core` = libraryDependencies ++= Seq(
     akkaActor,
     akkaStream,
+    play,
+    akkaPersistenceCassandraLauncher,
 
     // Upgrades needed to match whitelist
-    scalaJava8Compat,
-    scalaParserCombinators
```

Comment:
No longer require overrides because Play transitively depends on the right versions.